src/Compilers/CSharp/Portable/BoundTree/Constructors.cs

```diff
@@ -175,14 +175,14 @@ public static BoundCall Synthesized(SyntaxNode syntax, BoundExpression? receiver
             return Synthesized(syntax, receiverOpt, method, ImmutableArray.Create(arg0, arg1));
         }
 
-        public static BoundCall Synthesized(SyntaxNode syntax, BoundExpression? receiverOpt, MethodSymbol method, ImmutableArray<BoundExpression> arguments)
+        public static BoundCall Synthesized(SyntaxNode syntax, BoundExpression? receiverOpt, MethodSymbol method, ImmutableArray<BoundExpression> arguments, ImmutableArray<RefKind> argumentRefKindsOpt = default)
         {
             return new BoundCall(syntax,
                     receiverOpt,
                     method,
                     arguments,
                     argumentNamesOpt: default(ImmutableArray<string>),
-                    argumentRefKindsOpt: method.ParameterRefKinds,
+                    argumentRefKindsOpt: argumentRefKindsOpt.IsDefault ? method.ParameterRefKinds : argumentRefKindsOpt,
```

Comment:
>argumentRefKindsOpt [](http://example.com/codeflow?start=100&length=19)

Should probably assert that values are adjusted similar to how ```GetEffectiveArgumentRefKinds``` does that.

Reply:
Added an assert although this is also asserted in EmitArgument (but I guess it's harder to debug the root cause when that assert fails).